lib/note-detail.jsx

```diff
@@ -90,16 +114,12 @@ export default React.createClass( {
 	},
 
 	renderEditable( divStyle ) {
-		var { note } = this.props;
-		var valueLink = {
-			value: this.state.content,
-			requestChange: this.props.onChangeContent.bind( null, note )
-		};
+		const note = this.props.note;
 
 		return (
 			<Textarea ref={ this.initializeNoteEditor } className="note-detail-textarea theme-color-bg theme-color-fg"
 				disabled={ !!( note && note.data.deleted ) }
-				valueLink={ valueLink }
+				onChange={ this.onEditorChanged }
```

Comment:
given that we have a succinct helper to work with, we could actually eliminate `onEditorChanged` and make the call here:

```js
onChange={ debounce( this.saveNote, saveDelay, { maxWait: minimumSaveInterval } ) }
```